src/lib/media/video/errors.ts

```diff
@@ -1,6 +1,6 @@
 export class VideoTooLargeError extends Error {
   constructor() {
-    super('Videos cannot be larger than 100MB')
+    super('Videos cannot be larger than 50mb')
```

Comment:
```suggestion
    super('Videos cannot be larger than 50MB')
```
nit: I think the capitalised MB was correct as it's megabytes being referred to?

Reply:
Good call, lets do that.

Reply:
(thank you!!)

Reply:
Ffs, it didn't capitalize it lol. Opening a PR.

Reply:
Ahh, actually doesn't matter. That string doesn't get displayed to the user anyway.